ext/storage_sources/azure_store/test/test_azure_connection.cpp

```diff
@@ -31,4 +31,39 @@
 
 #include "azure_connection.h"
 
-TEST_CASE("testing class azure_connection", "azure_connection") {}
+TEST_CASE("testing class azure_connection", "azure_connection")
+{
+
+    std::vector<std::string> objects;
+    azure_connection pfx_test = azure_connection("myblobcontainer1", "pfx_test_");
+
+    // There is nothing in the container so there should be 0 objects
+    objects.clear();
+    REQUIRE(pfx_test.list_objects("", objects, false) == 0);
```

Comment:
better name would be `prefix_conn` rather than `pfx_test`

Reply:
nice suggestion - changed https://github.com/wiredtiger/wiredtiger/pull/8738/commits/62f7e4766feef2010b903baa8e78a7234e1274da